committer-tools/verify_license.py

```diff
@@ -77,14 +77,14 @@ def get_license_deps(license_text):
 def main():
     # Argument parser
     parser = argparse.ArgumentParser(description="Whether to skip executing ReleaseTarGz.")
-    parser.add_argument('--skipReleaseTarGz', type=str, default=False, help='Whether to skip ReleaseTarGz (true/false)')
+    parser.add_argument('--skip-build', action="store_true", help='Whether to skip ReleaseTarGz (true/false)')
```

Comment:
nit: mixture of single and double quotes. I'd suggest using double quotes like the rest of the file.

Also the `(true/false)` part is not necessary. The help text should say what setting this flag does, e.g. "skip the build"